apps/studio/src/components/TabShell.vue

```diff
@@ -0,0 +1,471 @@
+<template>
+  <div
+    class="query-editor"
+    ref="container"
+    v-hotkey="keymap"
+  >
+    <div
+      class="top-panel"
+      ref="topPanel"
+    >
+      <mongo-shell
```

Comment:
Any way to limit the characters in the prompt? I can imagine users having super=duper-long hostnames and database names, then the prompt is the whole width of the screen